IO/NativeSignalHandlers/WindowsNTHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Runtime.InteropServices;
+
+namespace LocalAdmin.V2.IO.NativeSignalHandlers
+{
+    /// <summary>
+    ///     Native signal processing on Windows NT systems.
+    /// </summary>
+    internal sealed class WindowsNTHandler : INativeSignalHandler
+    {
+        public static readonly WindowsNTHandler Handler;
+
+        static WindowsNTHandler()
```

Comment:
Initialize it in the variable declaration

Reply:
The same answer as here -> https://github.com/northwood-studios/LocalAdmin-V2/pull/4#discussion_r434347360

Reply:
Resolved in https://github.com/northwood-studios/LocalAdmin-V2/pull/4/commits/afc8b33f4dac4ece0e9b22d17701c63beea1854e